2-Browsers/Week1/homework/ex4-whatsTheTime/index.js

```diff
@@ -1,12 +1,32 @@
 'use strict';
+
+// const { Body } = require('node-fetch');
+
 /*------------------------------------------------------------------------------
 1. Inside the `index.js`, complete the `addCurrentTime` to add the current time 
   to the webpage. Make sure it's written in the HH:MM:SS notation (hour, minute,
   second). Use `setInterval()` to make sure the time stays current.
 2. Have the function execute when it's loading in the browser.
 ------------------------------------------------------------------------------*/
 function addCurrentTime() {
-  // TODO complete this function
+  let now = new Date(),
+    hours = now.getHours(),
+    minutes = now.getMinutes(),
+    seconds = now.getSeconds();
+
+  hours < 10 ? (hours = '0' + hours) : hours;
+  minutes < 10 ? (minutes = '0' + minutes) : minutes;
+  seconds < 10 ? (seconds = '0' + seconds) : seconds;
```

Comment:
great thinking! 💥 although.. normally ternaries are formatted a bit differently!

```suggestion
  let paddedHours = hours < 10 ? (hours = '0' + hours) : hours;
  let paddedMinutes = minutes < 10 ? (minutes = '0' + minutes) : minutes;
  let paddedSeconds = seconds < 10 ? (seconds = '0' + seconds) : seconds;
```

Reply:
Generally you want to use both cases of a ternary and not just do nothing in the `else`-case